### PR TITLE
header - overwrite logo with logo from datadir at runtime

### PR DESCRIPTION
**Pull request by Vampouille**
Should fix https://github.com/georchestra/georchestra/issues/1642

### CODE REVIEW

File: header/src/main/java/org/georchestra/_header/DefaultController.java [hunk 1]
```diff
@@ -35,7 +41,24 @@
...
+    private ServletContext context;
+
+    @Autowired
+    private GeorchestraConfiguration georchestraConfiguration;
+
+    @PostConstruct
+    public void replaceLogo() throws IOException {
+
+        // Try to replace logo with logo from datadir
+        File target = new File(this.context.getRealPath("/img"), "logo.png");
```
Comment (fvanderbiest) [inline header/src/main/java/org/georchestra/_header/DefaultController.java]: IIUC, this means it will look for `/etc/georchestra/header/img/logo.png` ?

Comment (fvanderbiest) [inline header/src/main/java/org/georchestra/_header/DefaultController.java]: hmm, rather /etc/georchestra/header/logo.png

Comment (Vampouille) [inline header/src/main/java/org/georchestra/_header/DefaultController.java]: I tested with `/etc/georchestra/header/logo.png` and it works

Comment (Vampouille) [inline header/src/main/java/org/georchestra/_header/DefaultController.java]: source path is computed on next line

**Comment by fvanderbiest** [comment]
OK, LGTM !

**Pull request merged**